### ISSUE TITLE
Build failure due to 403 Forbidden on getting Dynmap

### ISSUE THREAD

**Issue by ArcCha** [MiraWaNeko/DiscordIntegration#127]
### Base informations
**Minecraft version:** Not applicable.

**DiscordIntegration version:** 1.12 (commit: fa3ec0495fb111e6fb5c3ad0918dc4ea26e2af50)

**Link to pastebin with (censored) config:** *Optional*

### Overview

I am actively using this plugin (thanks!) and several users on my Discord Server where unhappy that each login/logout event is being posted to Discord chat. I know I can disable this, but they requested that the number of currently playing users should be put in the Discord channel topic, maybe with nicknames. So I've tried to modify this plugin to do so, but I failed pretty early - I've cloned the repo and tried to run `gradle build`, but it failed with

```
> Configure project : 
A constructor for `org.gradle.api.internal.java.JavaLibrary` is used by Shadow plugin v1.2.x, and has been preserved for compatibility. This has been deprecated and is scheduled to be removed in Gradle 5.0. If you're using the Shadow plugin, try upgrading to v2.x.
        at build_c6lqbi09bw50z8x9pljevjjme.run(/home/arccha/programowanie/DiscordIntegration/build.gradle:28)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
This mapping 'snapshot_20170624' was designed for MC 1.12! Use at your own peril.


FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':compileClasspath'.
> Could not resolve 2436/596:Dynmap:2.6-beta-1-forge-1.12.
  Required by:
      project :
   > Could not resolve 2436/596:Dynmap:2.6-beta-1-forge-1.12.
      > Could not get resource 'http://addons-origin.cursecdn.com/files/2436/596/Dynmap-2.6-beta-1-forge-1.12.xml%3Ftoken=0032bf10-b500-406c-90da-82737679be79'.
         > Could not GET 'http://addons-origin.cursecdn.com/files/2436/596/Dynmap-2.6-beta-1-forge-1.12.xml%3Ftoken=0032bf10-b500-406c-90da-82737679be79'. Received status code 403 from server: Forbidden
```

I've tried to google this error, thinking that it should be a simple matter of getting credentials from CurseForge, but I found nothing. Can you please guide me in the right direction?

**Comment by 132ikl** [comment]
The token probably expired. If I remember I could try to build it tonight
but if I get the same issue @Chikachi has to fix this

**Comment by CDAGaming** [comment]
I've removed Dynmap Integration in my fork until this is fixed :D

**Comment by MiraWaNeko** [comment]
For now, you can download the jar file from CurseForge, put it in the root of the project and rename it to Dynmap.jar

**Comment by ArcCha** [comment]
@CDAGaming thanks, I will use your fork, it's much more covenient for me.

**Comment by MiraWaNeko** [comment]
Fixed in 5d952bbb8c129f368c9f89bedbfec2201217efc6
